### ISSUE TITLE
Project configuration exception with Kotlin jvm/kapt 1.4.0 and Loom 0.5.25

### ISSUE THREAD

**Issue by ChloeDawn** [FabricMC/fabric-loom#273]
```
Caused by: java.lang.NoSuchFieldError: Companion
	at net.fabricmc.loom.util.mixin.KaptApInvoker.lambda$getConfigurations$1(KaptApInvoker.java:94)
	at net.fabricmc.loom.util.mixin.KaptApInvoker.getConfigurations(KaptApInvoker.java:96)
	at net.fabricmc.loom.util.mixin.KaptApInvoker.<init>(KaptApInvoker.java:51)
	at net.fabricmc.loom.AbstractPlugin.lambda$configureCompile$11(AbstractPlugin.java:353)
```
https://github.com/FabricMC/fabric-loom/blob/80aaf638324f19c6890a17bbec701f2f1bcbd82c/src/main/java/net/fabricmc/loom/util/mixin/KaptApInvoker.java#L90-L97

The function was moved here https://github.com/JetBrains/kotlin/blob/a766369e7283d7b89a5ca2960460dc20365f919a/libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/Kapt3KotlinGradleSubplugin.kt#L90

**Comment by modmuss50** [comment]
Could you provide an example buildscript? Thanks

**Comment by ChloeDawn** [comment]
Written in Kotlin as I use their DSL in my workspaces, however it can easily be translated to Groovy.
No `kapt` dependency notation is needed to produce the error, it occurs during plugin configuration.
```kotlin
plugins {
  id("fabric-loom") version "0.5.25"
  kotlin("jvm") version "1.4.0"
  kotlin("kapt") version "1.4.0"
}

group = "net.fabricmc"
version = "1.0.0"

java {
  sourceCompatibility = JavaVersion.VERSION_1_8
  targetCompatibility = sourceCompatibility
}

dependencies {
  minecraft("com.mojang:minecraft:1.16.2")
  mappings("net.fabricmc:yarn:1.16.2+build.47:v2")
  modImplementation("net.fabricmc:fabric-loader:0.9.3+build.207")
}
```

**Comment by nicholascioli** [comment]
In case anyone else has this issue, turns out it was caused by an out-of-date Kotlin plugin (template Kotlin fabric mod was using Kotlin version 1.4.0, my installed plugin was at version 1.3.72). I suspect this is because gradle doesn't actually download the plugin if run from an IDE?

These are the steps I took to fix it:
- Download newest compatible version from [here](https://plugins.jetbrains.com/plugin/6954-kotlin/versions)
- Update the installed plugin in your IDE (I was using Android Studio)
- Restart the IDE

Edit: Turns out this doesn't fix it either. Updating `settings.gradle` to the following seemed to fix the issue for now.

```properties
kotlin_version=1.3.72
fabric_kotlin_version=1.3.72+build.1
```
